apis/v1alpha1/tcproute_types.go

```diff
@@ -60,11 +60,28 @@ type TCPRouteStatus struct {
 
 // TCPRouteRule is the configuration for a given rule.
 type TCPRouteRule struct {
-	// Matches define conditions used for matching the rule against
-	// incoming TCP connections. Each match is independent, i.e. this
-	// rule will be matched if **any** one of the matches is satisfied.
-	// If unspecified, all requests from the associated gateway TCP
-	// listener will match.
+	// Matches define conditions used for matching the rule against incoming TCP
+	// connections. Each match is independent, i.e. this rule will be matched if
+	// **any** one of the matches is satisfied. If unspecified, all requests
+	// from the associated gateway listener will match.
+	//
+	// Each client request should map to a maximum of one route rule. If a
```

Comment:
nit: `s/should/must`?

Reply:
Strong +1 from me here, and on the other places where this phrase is used.

Reply:
Updated, thanks!